src/AzureAuth/CommandMain.cs

```diff
@@ -176,37 +174,17 @@ public CommandMain(CommandExecuteEventData eventData, ITelemetryService telemetr
         public double Timeout { get; set; } = GlobalTimeout.TotalMinutes;
 
         /// <summary>
-        /// Gets or sets the cache file name. Only available on Windows.
+        /// Gets the cache file name. Only available on Windows.
         /// </summary>
-        [Option(CacheOption, "Override the default cache file location. This option is only available on Windows.", CommandOptionType.SingleValue, ShowInHelpText = false)]
-        [LegalFilePath]
         public string CacheFilePath

```

Comment:
Since this isn't going to be configurable, we can further simplify things. Instead of leaving this property here and passing it down to the PCA Wrapper, the PCAWrapper can now own this path creation itself.